connector/rest/src/main/resources/config/ms-core-rest-client.properties

```diff
@@ -3,6 +3,25 @@ rest-client.ms-core.base-url=${MS_CORE_URL:http://localhost:8080}
 rest-client.ms-core.createPnPgInstitution.path=/external/institutions/pn-pg
 rest-client.ms-core.onboardingInfo.path=/users/{userId}/institution-products
 rest-client.ms-core.getToken.path=/tokens/token
+rest-client.party-process.getOnBoardingInfo.path=/onboarding/info
+rest-client.party-process.getInstitution.path=/institutions/{id}
+rest-client.party-process.getUserInstitutionRelationships.path=/institutions/{id}/relationships
+rest-client.party-process.verifyOnboardingByExternalId.path=/onboarding/institution/{externalId}/products/{productId}
+rest-client.party-process.verifyOnboarding.path=/onboarding
+rest-client.party-process.getInstitutions.path=/institutions/
+rest-client.party-process.createInstitutionFromIpa.path=/institutions/from-ipa/
+rest-client.party-process.createInstitutionFromPda.path=/institutions/from-pda/
+rest-client.party-process.createInstitutionFromAnac.path=/institutions/from-anac/
+rest-client.party-process.createInstitutionFromIvass.path=/institutions/from-ivass/
+rest-client.party-process.createInstitution.path=/institutions/
+rest-client.party-process.getInstitutionByExternalId.path=/external/institutions/{externalId}
+rest-client.party-process.createInstitutionUsingExternalId.path=/institutions/{externalId}
+rest-client.party-process.onboardingOrganization.path=/onboarding/institution/complete
+rest-client.party-process.createInstitutionRaw.path=/institutions/insert/{externalId}
+rest-client.party-process.getUserInstitutionRelationshipsByExternalId.path=/external/institutions/{externalId}/relationships
+rest-client.party-management.getRelationships.path=/relationships
+rest-client.party-management.findByGeoTaxonomies.path=/institutions/bygeotaxonomies
```

Comment:
you could also unify the id of the client using "rest-client.ms-core" instead "rest-client.party-process"